package.json

```diff
@@ -16,12 +16,12 @@
   },
   "dependencies": {
     "@changesets/cli": "^2.14.1",
-    "ethers": "^5.6.2",
+    "ethers": "5.6.2",
```

Comment:
Why did you 'downgrade' ethers version?

Reply:
To make usedapp predictable. Otherwise it just installs the latest version of ethers compatible with 5.6.2